tasks/agent6-linux.yml

```diff
@@ -10,6 +10,13 @@
     dest: /etc/datadog-agent
     state: directory
 
+- name: Create system-probe configuration file
+  template:
+    src: system-probe.yaml.j2
+    dest: /etc/datadog-agent/system-probe.yaml
+    owner: "root"
```

Comment:
Is there a situation where a user might want to change this ? Should we make it configurable ?

Reply:
Not to my knowledge but I suppose if someone is using linux capabilities, they may create a different user for this purpose.  It can't hurt to template it out.

Reply:
Well templating it means we support if officially, I let you choose if it's something we want to do.

Reply:
@sunhay Do we want this templated?